scripts/scr_apothecary_ship/scr_apothecary_ship.gml

```diff
@@ -68,36 +68,14 @@ function scr_apothecary_ship() {
 	// show_message("[1] "+string(obj_ini.ship[1])+": "+string(ship_apoth[1])+" apothecaries");
 
 
-	var normal_hp,mixhp,ratio;
+	var normal_hp,mixhp,ratio,unit, unit_location;
```

Comment:
`unit_location` is not used at all?
`unit` can be deleted from here, it's declared and used about 30 lines below

Reply:
just declared at top of page as otherwise it's just repeated declaration

Reply:
if it's not used, why is it there?